### DISCUSSION TITLE
Sorry, it was an issue with my settings. Please disregard it.

### DISCUSSION THREAD

**Discussion by xkia** [JetBrains/ideavim#941] [General]
In Vim mode, entering 'ga' inserts the character 'syAlign)' at the current configuration. Currently, the action mapped to 'ga' is set as follows: `noremap ga <ESC>:action AceWordAction<CR>`. Even if this mapping is commented out, it still inserts the character 'syAlign)' at the current position.

**Comment by xkia** [comment]
Sorry, it was an issue with my settings. Please disregard it.

**Reply by lippfi** [reply]
Offtopic:
It's strongly recommended to use the action notation in mappings e.g. 
`noremap ga <Action>(AceWordAction)`
it is faster, more consistent and safe (this mapping does not open ex input prompt in the bottom of the editor and calls the IDE action directly)
